app/pages/components/LibraryPage.js

```diff
@@ -0,0 +1,105 @@
+// @flow
+
+import * as React from 'react';
+import { translate } from 'react-i18next';
+import type { TranslatorProps } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import { Button, Card, Divider, Grid, Image } from 'semantic-ui-react';
+
+import Page from '../Page';
+
+type Props = TranslatorProps & { /* new props go here */ };
+
+const description1 = `This is an introductory course on the subject of mutations.
+                      It should help you understand how genetic evolution works
+                      and why mutants, like the famous X-men, exist`;
+
+const description2 = `This is an introductory course on the subject of modern slang.
+                      It should help the reader to better understand teens,
+                      as well as peoplez of the internet, and how they
+                      communicate. Beware: course contains dank memes`;
+
+const description3 = `This is an introductory course on the subject of DBZ
+                      and its cultural impact. Beware: this is a Work In Progress.
+                      Or better yet: rejoice, because this isn't even the topic's FINAL FORM!`;
+
+const LibraryPage = (props: Props): React.Node => {
+  const { t } = props;
+
+  return (
+    <Page>
+      <Grid.Row>
+        <Grid padded="vertically">
+          <Grid.Column width={12}>
+            <h1>{t('pages:librarypage.title')}</h1>
+          </Grid.Column>
+          <Grid.Column width={4}>
+            <Button floated="right">
+              <Link to="/topics/new">{t('common:link.newtopic')}</Link>
```

Comment:
Does `Link` rewrite absolute URLs?

Reply:
Link does not work if I run it on my computer.